src/main/java/lukemccon/airdrop/packages/Package.java

```diff
@@ -35,18 +42,18 @@ public double getPrice() {
 		return this.price;
 	}
 
+	public String getName() { return this.name; }
+
 	public Boolean canAfford(Player player) {
```

Comment:
This whole method could be shortened to one line ```return econ.getBalance(player) > this.price;```. Or it could be ```return Double.compare(econ.getBalance(player), this.price) >= 0``` to preserve the double compare if its due to floating point rounding.